be/src/olap/rowset/segment_v2/column_writer.cpp

```diff
@@ -262,6 +262,11 @@ Status ColumnWriter::create(const ColumnWriterOptions& opts, const TabletColumn*
         }
         case FieldType::OLAP_FIELD_TYPE_MAP: {
             DCHECK(column->get_subtype_count() == 2);
+            if (column->get_subtype_count() < 2) {
+                return Status::InvalidArgument(
+                        "If you upgraded from version 1.2.*, please drop the "
```

Comment:
return Status::InternalError to print stack, InvalidArgument will not print stack